isotovideo

```diff
@@ -157,40 +157,31 @@ sub stop_commands {
     return unless $cmd_srv_process->is_running;
 
     my $pid = $cmd_srv_process->pid;
-    diag("terminating command server $pid because $reason");
+    diag("stopping command server $pid because $reason");
 
-    # inform websocket clients
-    # note: If the job is stopped by the worker because it has been aborted, the worker will send this command
-    #       on its own to the command server and also stop the command server. So this is only done in the case
-    #       the test execution just ends.
     if ($cmd_srv_port && $reason && $reason eq 'test execution ended') {
         my $job_token = $bmwqemu::vars{JOBTOKEN};
         my $url       = "http://127.0.0.1:$cmd_srv_port/$job_token/broadcast";
         diag('isotovideo: informing websocket clients before stopping command server: ' . $url);
 
+        # note: If the job is stopped by the worker because it has been
+        # aborted, the worker will send this command on its own to the command
+        # server and also stop the command server. So this is only done in the
+        # case the test execution just ends.
+
         my $timeout = 15;
         my $ua      = Mojo::UserAgent->new(request_timeout => $timeout);
-        my $tx      = $ua->post($url, json => {stopping_test_execution => $reason});
         try {
-            my $res = $tx->result;
-            if ($res->code != 200) {
-                my $error_message = $res->to_string;
-                diag('isotovideo: unable to inform websocket clients about stopping command server: ' . $error_message);
-                if ($error_message =~ qr/.*timeout.*/i) {
-                    diag("isotovideo: The command server could be under heavy load and the timeout of $timeout seconds has been exceeded. This message is most likely not be related to the MAX_JOB_TIME of the current job possibly being exceeded.");
-                }
-                else {
-                    diag('The command server might have already been stopped by the worker after the user has aborted the job or the job timeout has been exceeded.');
-                }
-            }
+            $ua->post($url, json => {stopping_test_execution => $reason});
         }
         catch {
-            diag('isotovideo: unable to inform websocket clients about stopping command server: ' . $_);
+            # The command server might have already been stopped by the worker
```

Comment:
What exceptions do you expect to catch here? The comment doesn't make much sense for the code within the try.
Note that the try previously catched exceptions thrown by the result method. The mere post request is not supposed to throw exceptions.
I don't like that this code gives up on error handling completely and even supresses exceptions from the post method.
@kraih What do you think?

Reply:
you are right. I deleted the try…catch completely, even simpler :) Please check again. Maybe we are missing flows from the developer mode in our os-autoinst self-tests?

Reply:
Well, i never got to see the case in question. :)